python/composio/tools/env/host/workspace.py

```diff
@@ -84,6 +86,9 @@ def __init__(self, config: Config):
 
     def setup(self) -> None:
         """Setup workspace."""
+        self._is_ssh_client_set_up = False
```

Comment:
Consider moving this initialization to `__init__` method since it's a class attribute initialization. This would ensure the flag is always set when the instance is created.